frontend/src/views/LoginView.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import LoginView from './LoginView';
+
+describe('<LoginView />', () => {
+    var wrapper;
+    beforeEach(() => {
+        wrapper = shallow(<LoginView />);
+    });
+
+    describe('#handleSubmit()', () => {
+        it('should exist', () => {
+            expect(wrapper.handleSubmit).not.toEqual(undefined);
+        });
+
+        it('should be a function', () => {
+            expect(typeof(wrapper.handleSubmit)).toEqual('function');
+        });
+
+        it('should require one argument', () => {
+            expect(() => wrapper.handleSubmit()).toThrow(Error);
+        });
+
+        it('should call event.preventDefault()', () => {
+            const event = { preventDefault: jest.fn() };
+
+            wrapper.handleSubmit(event);
+
+            expect(event.preventDefault).toHaveBeenCalled();
+        });
+    });
```

Comment:
Excellent implementation of TDD - starting with a test for the existence of the method on the wrapper and proceeding from there based on the functionality we design into the method is absolutely the right way to go about fleshing out this test.